content/articles/testing-angular-app-with-protractor-and-jasmine/index.md

```diff
@@ -0,0 +1,243 @@
+
+
+# Testing Angular applications using Protractor and Jasmine
+![protractor](/engineering-education/testing-angular-app-with-protractor-and-jasmine/protractor.jpg)
+In this tutorial, we will be looking at writing a test for an Angular-enabled website created using protractor and jasmine. We will navigate to www.video.blender.org. Then we will first grab the angular enabled search input DOM elements using protractor API methods, then search for "free" videos by programmatically inputting the text we which to search then we will check the DOM interaction it causes. Based on our expectations we are bound to get one or more free videos available. The result causes an interactive change which will be our action to test for in our test case using jasmine.
```

Comment:
This intro looks like an abrupt start.

When you are introducing a new concept/technology, make sure to give an overview of what it is, why we use it, how it's better than the existing ones, then talk about what the tutorial is about, and the key takeaways

Reply:
Go through Section articles by [Ahmad](https://www.section.io/engineering-education/authors/ahmad-mardeni/) or [Louise](https://www.section.io/engineering-education/authors/louise-findlay/) to understand how their flow and structure is.